common/metrics/defs.go

```diff
@@ -2178,6 +2178,8 @@ const (
 
 	HashringViewIdentifier
 
+	AsyncRequestPayloadSize
```

Comment:
nit: move to the correct order although the ordering is already broken
```
ArchivalConfigFailures
ActiveClusterGauge
AsyncRequestPayloadSize
```

Reply:
It's not following an alphabetical order. The order is by functionality.